src/constants/global.js

```diff
@@ -0,0 +1 @@
+export const ESTIMATED_GAS_AVG = 135_000
```

Comment:
I'd maybe add this to a `src/constants/transactions.js` or something like that instead of a `global.js` (don't know @pmprete if this makes sense)

Reply:
Transactions sounds prettier, but im cool with both ways